ros/src/twist_controller/dbw_helper.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+#-------------------------------------------------------------------------------
+# Author: Kostas Oreopoulos <kostas.oreopoulos@gmail.com>
+# Date:   28.08.17
+#-------------------------------------------------------------------------------
+"""
+A collection of helper functions for dbw node
+"""
+# pylint: disable=import-error
+from math import sqrt, cos, sin
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
There is no place for this here. This requires xserver and won't run in headless mode when configured this way. Also, we're only supposed to use the dependencies that are in the requirements file and nothing else.

Common practice to do this sort of debugging is to dump data to a file and have a separate plotting script.